Submitted By: DJ Lucas <dj_At_linuxfromscratch_DOT_org>
Date: 2004-12-25
Origin: OOO-Build-1.3.7
Initial Package Version: 1.1.4
Description: Fixes build when used with gcc-3.3.4.
Upstream Status: Not required

$LastChangedBy$
$Date$

--- OOo_1.1.3_src-orig/icu/makefile.mk	2003-07-16 12:26:10.000000000 -0500
+++ OOo_1.1.3_src/icu/makefile.mk	2004-12-19 00:10:13.591321672 -0600
@@ -93,7 +93,7 @@
 
 CONFIGURE_DIR=source
 
-CONFIGURE_ACTION=sh -c 'CFLAGS=-O CXXFLAGS=-O ./configure --enable-layout --enable-static --enable-shared=yes --enable-64bit-libs=no'
+CONFIGURE_ACTION=sh -c 'CFLAGS=-O CXXFLAGS=-O ./configure --enable-layout --enable-static --enable-shared=yes --enable-64bit-libs=no --enable-extras=no'
 
 #CONFIGURE_FLAGS=--enable-layout --enable-static --enable-shared=yes --enable-64bit-libs=no
 CONFIGURE_FLAGS=
